Bottom: 8b422b5fa944de8f12ee34e1564151137c8838e5
Top:    eb0e4f33cbe4a11a5674055bc7ea895ae9de4da2
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-19 11:24:16 -0400

Support for AC97 on Phytec pmc030 base board.

Support for AC97 on Phytec pmc030 base board. This uses the generic fabric driver binding.

Signed-off-by: Jon Smirl <jonsmirl@gmail.com>


---

diff --git a/arch/powerpc/boot/dts/pcm030.dts b/arch/powerpc/boot/dts/pcm030.dts
index 8958347..001c8ef 100644
--- a/arch/powerpc/boot/dts/pcm030.dts
+++ b/arch/powerpc/boot/dts/pcm030.dts
@@ -190,6 +190,14 @@
 			cell-index = <0>;
 			reg = <0x2000 0x100>;
 			interrupts = <2 1 0>;
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			codec@0 {
+				compatible = "wlf,wm9712";
+				reg = <0x0>;
+			};
 		};
 
 		/* PSC2 port is used by CAN1/2 */
diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 266c6b7..11735e4 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -48,4 +48,11 @@ config SND_MPC52xx_SOC_EFIKA
 	help
 	  Say Y if you want to add support for sound on the Efika.
 
+config SND_MPC52xx_SOC_PCM030
+	tristate "SoC AC97 Audio support for Phytec pcm030 and WM9712"
+	depends on PPC_MPC5200_SIMPLE
+	select SND_SOC_MPC5200_AC97
+	select SND_SOC_WM9712
+	help
+	  Say Y if you want to add support for sound on the Phytec pcm030 baseboard.
